day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -134,14 +134,14 @@ joining an existing cluster.
 ====
 
 Back up the etcd data:
-+
+
```

Comment:
Not in the scope of this work but I find the admonition to be oddly placed and wonder if it would look better adding it under the super-task: 

> === Procedure
> Back up the etcd data:
> [NOTE]
> ====
> ...

Just a thought. But overall, LGTM!

Reply:
Not sure why GH made that quoted text ^^ so HUGE lol

Reply:
actually, i wasn't considering the IMPORTANT note you have after the task, so you can ignore my previous comment